### PR TITLE
Gemfile and completion output

### PR DESCRIPTION
**Pull request by arian-amador**
- Adds the Gemfile to go along with the ruby-gemset allowing users to just bundle install.
- Adjusted the completion output to print on a single line using the ANSI codes to move the cursor.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# A sample Gemfile
+source "https://rubygems.org"
+
+gem "chunky_png"
```
Comment (JVillella) [inline Gemfile]: This is great, thanks! Do you think we could specify the version? and also add `byebug` in there? I have the `require` commented out so it's easy for someone to quickly start debugging. I can update the `README.md` afterwards.

**Comment by JVillella** [comment]
I like the improvement with the console output. What do you think about doing it something along the lines of this:

```
diff --git a/tiny-path.rb b/tiny-path.rb
index 114eb74..309397c 100644
--- a/tiny-path.rb
+++ b/tiny-path.rb
@@ -8,6 +8,7 @@ Point = Col = Vec = Vector # alias
 EPSILON = 0.0001
 INVERTED_PI = 1.0 / Math::PI; TWO_PI = 2 * Math::PI
 MAX_DEPTH = 2
+CLEAR = "\r\e[K"

 class Numeric
   def clamp(min=0, max=1)
@@ -190,7 +191,8 @@ end
 def print_progress(current_pixel, pixel_count)
   if (current_pixel % (pixel_count / 100)) == 0
     percent = (current_pixel / pixel_count.to_f * 100).round(2)
-    puts "%3i%% complete" % percent
+    print CLEAR + "%3i%% complete" % percent
+    STDOUT.flush
   end
 end

@@ -240,4 +242,4 @@ PROGRESSIVE_SAVING = args[:prog_save]
 start_time = Time.now
 puts "Rendering #{args[:w]}x#{args[:h]}"
 save_image(render(args[:w], args[:h], args[:spp]), FILENAME)
-puts "Render completed in %.2f seconds" % (Time.now - start_time)
+puts CLEAR + "Render completed in %.2f seconds" % (Time.now - start_time)
```

Prints the progress on the same line as the cursor, and removes the need for `[A`. Constant `CLEAR` may help with readability if someone isn't used to the ANSI escape codes.

File: Gemfile (hunk 1 above)
Comment (arian-amador) [inline Gemfile]: Yup we can add it in there also and add version to both of them.

**Comment by arian-amador** [comment]
Agreed if it's being used in more than one line a constant makes sense. Looks good to me.

**Comment by JVillella** [comment]
Looks great! :+1:

**Pull request merged**